storybook/storybook-react/src/components/OrderedList/OrderedList.docs.mdx

```diff
@@ -1,15 +1,52 @@
-import { Canvas, Controls, Markdown, Meta, Primary } from "@storybook/blocks";
+import { Canvas, Meta } from "@storybook/blocks";
 import * as OrderedListStories from "./OrderedList.stories.tsx";
-import README from "../../../../../packages/css/src/ordered-list/README.md?raw";
 
 <Meta of={OrderedListStories} />
 
-<Markdown>{README}</Markdown>
+# Ordered list
 
-<Primary />
+Een geordende lijst is een verticale groep gerelateerde inhoud met een volgorde.
+
+## Voorbeelden
+
+### Basis
+
+De lijst heeft oplopende getallen als opsommingstekens.
+Er is genoeg ruimte voor getallen tot 99.
+Tussen de items staat extra witruimte.
+Dat geeft een beter onderscheid tussen de items, zeker als ze uit meerdere regels tekst bestaan.
+
+<Canvas of={OrderedListStories.Basis} />
 
 ### Twee niveaus
 
-<Canvas of={OrderedListStories.TwoLevels} />
+Een lijst mag één dieper niveau hebben.
+In dit geval worden onderkast letters gebruikt als opsommingsteken.
+
+<Canvas of={OrderedListStories.TweeNiveaus} />
+
+### Hoger startgetal
+
+Wanneer nodig kan de opsomming beginnen met een hoger getal.
+Gebruik het HTML-attribuut `start`.
+
+<Canvas of={OrderedListStories.Startgetal} />
+
+### Omgekeerde volgorde
+
+Ook kunnen de cijfers achterstevoren lopen.
+Gebruik het HTML-attribuut `reversed`.
+
+<Canvas of={OrderedListStories.AflopendeNummering} />
+
+### Lijst zonder opsommingstekens
+
+Een overzicht van artikelen of zoekresultaten is ook een geordende lijst.
+Gebruik in dit soort gevallen een lijst zonder opsommingstekens om die semantiek aan te brengen.
+Daardoor kunnen schermlezers helpen de lijst te begrijpen en te navigeren.
+
+In dit geval verwacht de lijst dat componenten in de items de typografie definiëren.
```

Comment:
This sentence is a bit tricky to parse. Maybe something like this?

```suggestion
Let op: deze lijst heeft geen stijling (zoals typografie en dergelijke) voor zijn items. Deze stijling moet je in de componenten in de items definiëren.
```

Reply:
Fair point. I did choose still another two sentences though.